proto/kvrpcpb.proto

```diff
@@ -208,8 +208,7 @@ message CheckTxnStatusResponse {
     uint64 commit_version = 4;
     // The action performed by TiKV (and why if the action is to rollback).
     Action action = 5;
-    bool use_async_commit = 6;
-    repeated bytes secondaries = 7;
+    LockInfo lock_info = 6;
```

Comment:
Reusing the same tag is a breaking change.

Reply:
It's ok since it's only on master. But I'm afraid if other projects need to struggle to get them compiled.